### PR TITLE
import/export json file

### LINKED ISSUE #1: Import / export to JSON file
Hey,
Thanks for the cool project. One uncomfortable thing that came out when using it is having to manually enter data if you want to update the map. It is nice, that there is a browser cache, but it is not very practical when changing computers, browsers or just clearing the cache. 

I suggest importing to JSON file. I had a quick look at the code and since there is already an established pattern of creating cache in localstorage there is no point in changing it, just allow to export it at any time and overwrite the current state with a JSON file. 

If you are open to this feature I can open a pull request in my spare time.

Cheers

### PR DESCRIPTION
**Pull request by ojurkiewicz**
Resolves #1

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -342,6 +353,32 @@ <h3 class="opacity-75 inline">wyniki wyszukiwania</h3>
           },
           showOptions () {
             this.options = !this.options
+          },
+          saveCacheToFile() {
+
```
Comment (krzysztofrewak) [inline index.html]: Please remove this unnecessary empty line.

File: index.html [hunk 2]
```diff
@@ -342,6 +353,32 @@ <h3 class="opacity-75 inline">wyniki wyszukiwania</h3>
...
+            const cache = localStorage.getItem('cached')
+            const fileName = 'powiaty.json'
+
+            const element = document.createElement('a');
```
Comment (krzysztofrewak) [inline index.html]: I'm not using semicolons in this project. I know that I should provide some linter for that, but I didn't know that anyone would like to contribute to this project. ;)

File: index.html [hunk 3]
```diff
@@ -342,6 +353,32 @@ <h3 class="opacity-75 inline">wyniki wyszukiwania</h3>
...
+
+            element.style.display = 'none';
+            document.body.appendChild(element);
+
+            element.click();
+
+            document.body.removeChild(element);
+          },
+          pickFile() {
+            document.getElementById("fileUpload").click()
```
Comment (krzysztofrewak) [inline index.html]: The same is with doublequotes; please change them to single ones,

**Review by krzysztofrewak — CHANGES_REQUESTED** [review]
Good job, but I need to ask you for som eminor fixes.

There's also a minor UI glitch: icons aren't centered as they were before your changes. I'm using Firefox:
image

And sorry for long waiting time. I was on vacations. ;)

**Comment by ojurkiewicz** [comment]
Hey,
Sorry for lack of attention. Prettier made me lazy. 

As for the icons, this is not the result of my changes. Adding new buttons only exaggerated the problem. I am attaching Firefox and Chrome screenshots from the currently deployed version. I think you can see on them quite clearly that the icons are not in the center. For this reason, I would treat this as a separate issue. 

Cheers!

Screenshot 2021-06-15 at 19 22 22
Screenshot 2021-06-15 at 19 19 58

**Pull request merged**